tmt/base.py

```diff
@@ -4007,7 +4010,7 @@ def runs(self, id_: tuple[str, ...]) -> bool:
         if keep is not None:
             # Sort by modify time of the workdirs and keep the newest workdirs
             all_workdirs.sort(
-                key=lambda workdir: os.path.getmtime(workdir / 'run.yaml'), reverse=True)
+                key=lambda workdir: (workdir / 'run.yaml').stat().st_mtime, reverse=True)
```

Comment:
same as above: stat() vs lstat(), assuming we don't need to handle symlinks?

Reply:
`stat()` seems correct to me, we do want to follow symlinks - getting mtime of a symlink itself does not look useful.